corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_module_overwrite.html

```diff
@@ -70,15 +70,15 @@
           <br>
           <div class="row">
             <div class="col-sm-4 col-sm-offset-1">
-              <button class='btn btn-danger' data-toggle="modal" data-target="#overwrite-confirmation-model" type="button">
+              <button id="{{ detail_type }}" class='btn btn-danger' data-toggle="modal" data-target="#{{ detail_type }}-overwrite-confirmation-model" type="button">
```

Comment:
I understand why you needed to add the prefix to `data-target`, but why do you need to add an id?

Reply:
my bad, removed it.